pom.xml

```diff
@@ -25,10 +25,10 @@
 
         <wildfly.groupId>org.wildfly</wildfly.groupId>
         <wildfly.artifactId>wildfly-dist</wildfly.artifactId>
-        <version.wildfly>10.1.0.Final</version.wildfly>
+        <version.wildfly>11.0.0.Final</version.wildfly>
         <wildfly.directory>wildfly-${version.wildfly}</wildfly.directory>
 
-        <version.wildfly.maven.plugin>1.1.0.Final</version.wildfly.maven.plugin>
+        <version.wildfly.maven.plugin>1.2.1.Final</version.wildfly.maven.plugin>
```

Comment:
I think it's good to update this version as close as possible to version `1.2.2.Final` when we'll have the fix for [WFMP-94](https://issues.jboss.org/browse/WFMP-94)

Reply:
+1 - Hopefully that gets released soon as well.